datastructures/pieces/Rook.cpp

```diff
@@ -23,3 +23,11 @@ std::vector<Position> Rook::possibleMoves() {
 char Rook::getCLIToken() {
     return 'R';
 }
+
+char32_t Rook::getUnicodeToken() {
+    if (player == WHITE) {
```

Comment:
Going to flip all of these as @jacobrs mentioned it would look better and more intuitive.